src/core/include/openvino/pass/pattern/op/optional.hpp

```diff
@@ -94,8 +94,10 @@ std::shared_ptr<Node> optional(const OutputVector& inputs, const TPredicate& pre
         attrs.empty() ? op::Predicate(pred) : attrs_match(attrs) && op::Predicate(pred));
 }
 
-template <class... NodeTypes, typename TPredicate>
-std::shared_ptr<Node> optional(const Output<Node>& input, const TPredicate& pred, const Attributes& attrs = {}) {
+template <class... NodeTypes, typename TPredicate = nullptr_t>
+std::shared_ptr<Node> optional(const Output<Node>& input,
+                               const TPredicate& pred = nullptr,
```

Comment:
Myabe is better to remove this parameter at all and create some default (null/always true) predicate?
`const TPredicate& pred = nullptr` looks strange as reference holds null pointer.